script2.js

```diff
@@ -1,15 +1,15 @@
 $(document).ready(function() {
   $('#addTask').click(function() {
-    let $name = $('#usName').val();
+    let $names = $('#usName').val();
     let $emp = $('#eid').val();
     let $city = $('#c_name').val();
-    let $checks = validate($name, $city, $emp);
+    let $checks = validate($names, $city, $emp);
     if ($checks === true) {
       let $res = $('<tr>', {
```

Comment:
is there any specific reason that you have used let and not var?

Reply:
since I'm not using the variable outside the function, used let(whose scope end at enclosing block)